HACKING.rst

```diff
@@ -86,18 +86,129 @@ You can also see all the environments by simply running ``tox list``
 Running ``tox run -m format`` and ``tox run -m lint`` before committing code is
 recommended.
 
-Code Style
```

Comment:
Should we keep this?

Reply:
No preference.  I changed the headings primarily because there are already 5 layers of headings and secondarily because I don't consider commit messages as code.

Reply:
No preference. I changed the headings primarily because there are already 5 layers of headings and secondarily because I don't consider commit messages as code.

Reply:
Yeah let's axe it. It was initially there as a placeholder to remind me to talk about our code style, but it seems a bit superfluous at this point since our style is basically "use our tooling and maybe do an adjustment run if it does something weird."